### DISCUSSION TITLE
Add PLAN mode to NDs

### DISCUSSION THREAD

**Discussion by KBNOtaku** [magknight/787aviatorsTracker#647] [Ideas]
**Describe the bug**
PLAN mode is not working.
I can not change and use PLAN mode anymore. I can not change into PLAN mode via the knob or the ND Touchscreen.
When I change into PLAN mode vie key, STEP is not working?!

**To Reproduce**
Use PLAN mode and check the route.

**Expected behaviour**
PLAN mode be able to use, it worked fine in the last stable version.

**Desktop (please complete the following information):**
 - OS:  Windows 10 ]
 - X-Plane version [ 11.55r2 ]
 - Version [ 1.7.18-alpha6 ]

**Comment by aeroniemi** [comment]
it's not planned to be added in the near future

**Reply by KBNOtaku** [reply]
But it still works with the pop out FMC?
So can´t you make it work like it was before?
Its an mandatory function to check the flight route anyway!?

**Reply by Simleybob** [reply]
People are going to get ‘angry’ if the plan function isn’t available in the rc, just a heads up.

**Reply by aeroniemi** [reply]
It didn't work to a useful extent before, so not having it at all now isn't a feature loss, especially now that you have longer-range ND views. (tbf, even if we wanted to add it, you'd either have it with no line, or it'd be the only available ND mode, which would be a massive feature loss)

**Reply by KBNOtaku** [reply]
I don't know what you mean, but PLAN mode does work perfectly fine in the stable version!
Longer ND Range doesn't help me to check my route when for example, the route is behind the Aircraft, or for an Long Haul etc..!
I agree with Simleybob, I also think a lot of people will get angry if PLAN mode isn't in RC version!

**Reply by Simleybob** [reply]
Yup, trust us aero.

**Reply by aeroniemi** [reply]
so, you'd prefer to have the old ND and plan mode, over the new ND?

**Reply by Simleybob** [reply]
Never said that

**Reply by KBNOtaku** [reply]
That sounds a little bit rude and like you are pissed? Sorry if I get you angry about that!
I am just asking to keep the PLAN mode. I don't think it will hurt the MAG787 if you let it as it is, but I thing it will hurt if you remove it?

Just an Update with ALPHA-10:
When now switching into PLAN mode via key binding, I am now again able to use STEP in the CDU, before I only could use it with popout FMC, thanks! Could be because of the force redraw you changed for it.
So the thing I am asking would be great if we pleas still could activate PLAN mode again with the knob or within the Menu in the ND?

**Reply by aeroniemi** [reply]
like it or not, that is the situation - there is no way to make it work effectively with the new ND. Simply put, if you want to use PLAN, stick with 1.7.18 that is currently on the live branch.

**Comment by KBNOtaku** [comment]
This now should be named "Possibility to activate PLAN mode again" or "Add PLAN mode again"

**Comment by Simleybob** [comment]
could it be worth bringing back the default ND to have Plan mode. since this is a thing in xp12? https://developer.x-plane.com/article/navigation-display-scaling/
